utils/env_preferences/src/error.rs

```diff
@@ -28,6 +31,10 @@ pub enum RetrievalError {
     #[cfg(target_os = "windows")]
```

Comment:
^`UnknownCategory` can also be linux-only?

Reply:
Good point!